lib/xandra/frame.ex

```diff
@@ -63,5 +67,24 @@ defmodule Xandra.Frame do
   defp encode_flags(_, false), do: 0x01
   defp encode_flags(_, true), do: 0x03
 
-  defp maybe_compress_body(nil, body), do: body
+  defp flag_set?(flags, flag) do
+    (flags &&& flag) > 0
```

Comment:
I won't use `band/2` as we use `|||` in `Protocol`, let's either change both or leave both like this. Also, `== 1` is broken as `0x03 &&& 0x02 == 0x02`, it could rather be `(flags &&& flag) == flag` but it reads odd to me.